acceptance/tests/concurrency/ticket_2659_concurrent_catalog_requests.rb

```diff
@@ -1,5 +1,9 @@
 test_name "concurrent catalog requests (PUP-2659)"
 
+tag 'audit:low',
+    'audit:integration',  # Testing server
+    'server'
```

Comment:
The comment is intended to indication that the test should be in the puppetserver integration tests. This information is covered by the tags themselves. I will delete the comment.